File: 1008 - Salary/Main.c
#include <stdio.h>

int main()
{
    int num, hpm;
    double mph;
    scanf("%d %d %lf", &num, &hpm, &mph);
    printf("NUMBER = %d\n", num);
    printf("SALARY = U$ %0.2lf\n", hpm * mph);
    return 0;
}

File: 1008 - Salary/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int num = scan.nextInt();
        int hpm = scan.nextInt();
        double mph = scan.nextDouble();
        System.out.printf("NUMBER = %d\n", num);
        System.out.printf("SALARY = U$ %.2f\n", hpm * mph);
        scan.close();
    }
}